indexer/indexer_tests/drules_selector_parser_test.cpp

```diff
@@ -16,6 +16,16 @@ UNIT_TEST(TestDruleSelectorIsSet)
   TEST_EQUAL(SelectorOperatorIsSet, e.m_operator, ());
 }
 
+UNIT_TEST(TestDruleSelectorIsSet2)
+{
+  SelectorExpression e;
+  TEST(ParseSelector("envelope_area", e), ());
```

Comment:
I suggest to use limit_rect - envelope_area is a complete mystery and hard to understand.